src/Component/Tests/Integration/Filter/Storage/SessionFilterStorageTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Tests\Integration\Filter\Storage;
+
+use Sylius\Component\Grid\Filter\Storage\FilterStorageInterface;
+use Sylius\Component\Grid\Symfony\Session\Filter\Storage\SessionFilterStorage;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+final class SessionFilterStorageTest extends KernelTestCase
```

Comment:
Does it mean each DI service in the project needs to be tested eventually? :eyes:

Reply:
Just testing the DI is minimalist, but testing with real DI is interesting. Cause you will be able to test it with the session in real condition.

Reply:
My question was more in "do we need it?" dimension, because if we make this a standard, then tests codebase will double with tests like that `$this->assertTrue($container->has('sylius_grid.filter_storage'));`.

Shouldn't SF/DI tests cover this?